client/modules/Registration/Registration.js

```diff
@@ -3,33 +3,113 @@ import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 import styles from './Registration.css'
 
+var validator = require('validator');
+
 //Create a component class
+
+function validate(email, password1, password2, mobile, license) {
+    // true means invalid, so our conditions got reversed
+    return {
+      email: !validator.isEmail(email),
+      password1: password1.length === 0,
```

Comment:
can we get a better password rule here? length > 0 is a bit relaxed :)

maybe use a regex test
this is an example for testing that it is at least 8 characters long, has an upper case, a lowercase and a number
"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$"